pyproject.toml

```diff
@@ -57,6 +58,29 @@ pyink-use-majority-quotes = true
 
 [tool.setuptools_scm]
 
+[tool.pytest.ini_options]
```

Comment:
I think this config or the CI config needs to be updated in order to ensure tests run properly.